src/Stratis.Bitcoin.Features.BlockStore/BlockStoreCache.cs

```diff
@@ -34,18 +37,50 @@ public class BlockStoreCache : IBlockStoreCache
         /// <summary>Provider of time functions.</summary>
         protected readonly IDateTimeProvider dateTimeProvider;
 
+        /// <summary>The blocks count that cache can contain.</summary>
```

Comment:
"The block count the cache can contain."

Reply:
Fixed